cmd/translatesfx/translatesfx/otel.go

```diff
@@ -25,6 +25,8 @@ type otelCfg struct {
 	Service       map[string]interface{}
 }
 
+const processlist = "smartagent/processlist"
```

Comment:
This should also include `kubernetes-events` but that can be another PR.  Some others should be in both metrics and logs (https://github.com/signalfx/splunk-otel-collector/blob/main/internal/receiver/smartagentreceiver/README.md#configuration  item 6)